pkg/services/ldap/ldap_test.go

```diff
@@ -247,6 +253,12 @@ func TestServer_Users(t *testing.T) {
 				{Name: "member", Values: []string{grootDN}},
 			}}},
 		}
+		babyCreaturesDN := "dn=babycreatures," + groupsOU
```

Comment:
:rofl: babycreatures

Reply:
Haha, yeah, like baby groot, baby yoda etc. Not ready to be Grafana Server Admins yet 😄